<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Danilo Martins Franco</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>2168</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="m_tabWidget_Main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>1071</width>
      <height>2126</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="m_Tab_Escolher">
     <attribute name="title">
      <string>Escolher</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>1050</width>
        <height>960</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #dddddd;
border-radius: 8px;
color: #444444;
font: 20pt &quot;Ubuntu Condensed&quot;;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
      </property>
      <widget class="QToolButton" name="m_toolButton_AdicionarEscolher">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>790</y>
         <width>330</width>
         <height>105</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #555555;
border-radius: 8px;
color: white;
font: 18pt &quot;Ubuntu Condensed&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>315</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
border-radius: 10px;
color: #444444;
text-decoration: none;
margin: 0px 0px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>Item</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
border-radius: 10px;
color: #444444;
text-decoration: none;
margin: 0px 0px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>Local</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>530</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
border-radius: 10px;
color: #444444;
text-decoration: none;
margin: 0px 0px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>R$</string>
       </property>
      </widget>
      <widget class="QComboBox" name="m_ComboBox_Local">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>1005</width>
         <height>140</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #fafa73;
border-radius: 8px;
color: black;
font: 18pt &quot;Ubuntu Thin&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
      <widget class="QComboBox" name="m_ComboBox_Item">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>365</y>
         <width>1005</width>
         <height>140</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #3ab7c9;
border-radius: 8px;
color: black;
font: 18pt &quot;Ubuntu Thin&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="m_ComboBox_Price">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>575</y>
         <width>1005</width>
         <height>140</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #fa7373;
border-radius: 8px;
color: black;
font: 18pt &quot;Ubuntu Thin&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
      </widget>
      <widget class="QLabel" name="m_label_DiaMesAno_Escolher">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>20</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Condensed&quot;;
border-radius: 10px;
color: #444444;
text-decoration: none;
margin: 0px 0px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="m_Tab_Editar">
     <attribute name="title">
      <string>Editar</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>1050</width>
        <height>960</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #dddddd;
border-radius: 8px;
color: #444444;
font: 20pt &quot;Ubuntu Condensed&quot;;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
      </property>
      <widget class="QTextEdit" name="m_textEdit_Local">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>1005</width>
         <height>140</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Thin</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #fafa73;
border-radius: 8px;
color: black;
font: 18pt &quot;Ubuntu Thin&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="m_textEdit_Item">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>365</y>
         <width>1005</width>
         <height>140</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Thin</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #3ab7c9;
border-radius: 8px;
color: black;
font: 18pt &quot;Ubuntu Thin&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhLatinOnly</set>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="m_textEdit_Preco">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>575</y>
         <width>1005</width>
         <height>140</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Thin</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #fa7373;
border-radius: 8px;
color: black;
font: 18pt &quot;Ubuntu Thin&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="m_toolButton_Adicionar">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>790</y>
         <width>330</width>
         <height>105</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #555555;
border-radius: 8px;
color: white;
font: 18pt &quot;Ubuntu Condensed&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>315</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
border-radius: 8px;
color: #444444;
text-decoration: none;
margin: 0px 0px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>Item</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
border-radius: 8px;
color: #444444;
text-decoration: none;
margin: 0px 0px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>Local</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>530</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
border-radius: 8px;
color: #444444;
text-decoration: none;
margin: 0px 0px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>R$</string>
       </property>
      </widget>
      <widget class="QLabel" name="m_label_DiaMesAno">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>20</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Condensed&quot;;
border-radius: 10px;
color: #444444;
text-decoration: none;
margin: 0px 0px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="m_Tab_Hoje">
     <attribute name="title">
      <string>Filtrar</string>
     </attribute>
     <widget class="QFrame" name="m_Frame_Hoje">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1056</width>
        <height>2071</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #dddddd;
border-radius: 8px;
color: #444444;
font: 14pt &quot;Ubuntu Condensed&quot;;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
      </property>
      <widget class="QTableView" name="m_tableView_Today">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>1036</width>
         <height>1141</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>1031</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignTop">
           <widget class="QLabel" name="m_LabelItem_HL">
            <property name="text">
             <string>Item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_ComboBoxItem_HL">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">width:70%;
height:80%;
background-color: #3ab7c9;
border-radius: 8px;
color: black;
font: 16pt &quot;Ubuntu Thin&quot;;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignTop">
           <widget class="QLabel" name="m_LabelPrice_HL">
            <property name="text">
             <string>Preços</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_ComboBoxPrice_HL">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">width:70%;
height:80%;
background-color: #fa7373;
border-radius: 8px;
color: black;
font: 16pt &quot;Ubuntu Thin&quot;;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignTop">
           <widget class="QLabel" name="m_LabelDate_HL">
            <property name="text">
             <string>Dia</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="m_dateEdit_HL">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">width:70%;
height:80%;
background-color: #ca7373;
border-radius: 8px;
color: black;
font: 16pt &quot;Ubuntu Thin&quot;;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
            </property>
            <property name="minimumDate">
             <date>
              <year>2020</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>dd-MM-yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignTop">
           <widget class="QLabel" name="m_LabelLocal_HL">
            <property name="text">
             <string>Local</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_ComboBoxLocal_HL">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">width: 40%;
height:80%;
background-color: #fafa73;
border-radius: 8px;
color: black;
font: 16pt &quot;Ubuntu Thin&quot;;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="m_pushButton_Filtrar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>1390</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40%;
background-color: #555555;
border-radius: 8px;
color: white;
font: 18pt &quot;Ubuntu Condensed&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
       </property>
       <property name="text">
        <string>Filtrar</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>1485</y>
         <width>296</width>
         <height>86</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">width: 40%;
background-color: #555555;
border-radius: 8px;
color: white;
font: 18pt &quot;Ubuntu Condensed&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
          </property>
          <property name="text">
           <string>Total:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">width: 40%;
background-color: #555555;
border-radius: 8px;
color: white;
font: 18pt &quot;Ubuntu Condensed&quot;;
padding: 16px 32px;
text-decoration: none;
margin: 4px 2px;
cursor: pointer;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>m_textEdit_Local</tabstop>
  <tabstop>m_textEdit_Item</tabstop>
  <tabstop>m_textEdit_Preco</tabstop>
  <tabstop>m_toolButton_Adicionar</tabstop>
  <tabstop>m_tabWidget_Main</tabstop>
  <tabstop>m_tableView_Today</tabstop>
  <tabstop>m_ComboBox_Item</tabstop>
  <tabstop>m_ComboBox_Local</tabstop>
  <tabstop>m_toolButton_AdicionarEscolher</tabstop>
  <tabstop>m_ComboBox_Price</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
